lib/src/main/java/space/taran/arklib/domain/meta/MetadataGenerator.kt

```diff
@@ -31,17 +31,16 @@ interface MetadataGenerator {
             val generator = GENERATORS.find {
                 it.isValid(path)
             } ?: let {
-                return Result.failure(
-                    IllegalArgumentException("No generators found for $path")
-                )
+                Log.d(LOG_PREFIX, "No generators found for $path")
+                return Result.success(Metadata.Unknown())
```

Comment:
Am I right, if we add new kinds later, we could just invalidate all existing metadata to generate metadata for what is `UNKNOWN` now?

Reply:
Yep, we can do it